prealloc.go

```diff
@@ -198,11 +198,22 @@ func exists(filename string) bool {
 	return err == nil
 }
 
+func contains(slice []string, item string) bool {
+	for _, s := range slice {
+		if s == item {
+			return true
+		}
+	}
+
+	return false
+}
+
 func (v *returnsVisitor) Visit(node ast.Node) ast.Visitor {
 
 	v.sliceDeclarations = nil
 	v.preallocMsgs = nil
 	v.returnsInsideOfLoop = false
+	var arrayTypes []string = nil
```

Comment:
the zero value of a slice is `nil`, so this can just be `var arrayTypes []string`

Reply:
👍

Reply:
sure